mlocal/frags/Makefile.stub

```diff
@@ -31,18 +31,21 @@ go_BIN := $(singularity)
 go_OBJ := $(SOURCEDIR)/src/pkg/buildcfg/config.go
 go_INSTALL := $(singularity_INSTALL)
 
+bash_completion :=  $(BUILDDIR)/etc/bash_completion.d/singularity
+bash_completion_INSTALL := $(PREFIX)/etc/bash_completion.d/singularity
```

Comment:
Need to remove `$(PREFIX)` here afaik so that the completion file just gets installed in  `/etc/bash_completion.d/singularity`.  Otherwise it won't be sourced when a new bash session is started.

Reply:
*kicks @GodloveD *

Reply:
I think we determined that installing everything in $(PREFIX) made for a cleaner and predictable install, where people interested in sourcing the completion file can easily throw that in their bash profile entries. I'd like to see some info around that though, but it seems like the better way to approach it. Rarely would software configured with --prefix should install stuff outside of that setup prefix.

Reply:
Blech.  If you insist I guess. What about making a `make completion` option or some such?

Reply:
As discussed, this will be handled via documentation.

Reply:
Boo!  😸